src/index.ts

```diff
@@ -11,10 +11,15 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import * as logging from 'plylog';
+import * as jsonschema from 'jsonschema';
+
 import minimatchAll = require('minimatch-all');
 
 const logger = logging.getLogger('polymer-project-config');
 
+const schema = JSON.parse(
```

Comment:
Let's try to not do sync file IO during module initialization. Add a `getSchema()` function that can lazily load it. It'd be even better if we avoided sync IO.